config/manager/kustomization.yaml

```diff
@@ -13,4 +13,4 @@ kind: Kustomization
 images:
 - name: perconalab/percona-server-mysql-operator
   newName: perconalab/percona-server-mysql-operator
-  newTag: main
+  newTag: k8sps-23
```

Comment:
please revert this and also tags in `deploy` dir, you can revert all of these by running `make generate manifests VERSION=main`